app/models/congress_member.rb

```diff
@@ -313,6 +340,15 @@ def fill_out_form_with_capybara f={}, driver
     end
   end
 
+  def clear_iframes(session)
```

Comment:
This is not very pretty, but it's a reliable workaround for a poltergeist bug where capybara elements won't be assigned ids if you set a custom user agent. Without element IDs, the only way we can access iframes is by their index number (which is also very unreliable and prone to timeouts). 

We're running a js script that removes all other iframes beside the recaptcha, allowing us to call within_iframe(0), 0 being the index for the only iframe on the page.